recipe/build.sh

```diff
@@ -3,6 +3,10 @@
 mkdir -p ${PREFIX}/lib/cmake
 mkdir -p ${PREFIX}/include
 
+pushd cccl-${PKG_VERSION}
```

Comment:
Weird, for some reason when running `python build-locally.py`, the source is put in this directory. However on CI (or if the Docker image is spun up manually and the build is done in there), the source directory is not named this way (and is not nested)